### ISSUE TITLE
navigation - it isn't clear where you are

### ISSUE THREAD

**Issue by charlottesales** [kentcc/kelsi#52]
Hi Allan, 

I know we are going to do some user testing on the navigation once we have more content in place, but I'm really struggling with using the nav - and this worries me as I've been involved with this from the start and I think users who are new to site aren't going to be any better. 

Once you get deeper into the navigation, there are no colour differentiations between levels and it's really hard to tell where you need to click without going back up a level. I know there are indents to help the user, but I don't think this is enough. 

Have you got any ideas of what we can do to improve it? Maybe more colours? Maybe closing the levels off as you go deeper? 

Thanks

**Comment by wellhairy** [comment]
Actually initially the navigation was designed so the levels close of as you go deeper. The navigation should go up to 3 levels, past this i don't think it's been coded to delineate the colour any further.

Lidor is doing some work to change the colour of the "current" navigation items after a conversation he and I had yesterday, This should help to inform users where they are. 

This may not be enough so another suggestion we can talk about after we've separated the colour from the current level and the last is possibly having a arrow on the current & last level denoting if your on a page of if your looking down the tree.

I think what we talked about is the navigation should go down three levels and the splinter off to a landing page however as it stands I don't think the content has been placed in the nav structure in this way.

Im welcome to any other suggestion you think of wile your putting in the content. I
